tests/unit/phpunit/modules/AOR_Reports/AOR_ReportTest.php

```diff
@@ -244,12 +244,13 @@ public function testbuild_report_html()
         $state->popGlobals();
     }
 
-    public function testgetTotalHTML()
+    public function testGetTotalHTML()
     {
         //execute the method with required data preset and verify it returns expected result
         $fields = array('label' => array('display' => 1, 'total' => 'SUM', 'label' => 'total'));
         $totals = array('label' => array(10, 20, 30));
-        $expected = "<table><thead class='fc-head'><tr><th>total Sum</th></tr></thead><tbody><tr class='oddListRowS1'><td>60</td></tr></tbody></table>";
+        $expected = "<table width='100%' class='list view table-responsive aor_reports'><thead class='fc-head'><tr><th>total Sum</th></tr></thead><tbody><tr class='oddListRowS1'><td>60</td></tr></tbody></table>";
```

Comment:
It looks like the tests are failing because they expect these to be double quotes rather than single quotes?